lib/diagnostic.rb

```diff
@@ -13,28 +17,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    sum = 0;
+    enum.foreach() do |num|
```

Comment:
The better answer for this question is to use `.reduce`:
`enum.reduce { |memo, num| memo + num }` (which could also be written `enum.reduce(&:+)`)